### PR TITLE
Add Facebook Formatting Helpers

### PR DESCRIPTION
**Pull request by byxor**
`formatting.py` contains helpers that apply text formatting.

✅ `code_block`
✅ `bold`
✅ `italic`
✅ `strikethrough`
✅ `monospace`

It's tested.

I went through a bunch of plugins and changed the duplicate code to use these helpers.
I may have missed some things.
I tried very hard to keep the plugins working. If they had tests, it'd be easy to catch errors quickly.

From now on, please do all your formatting through this library. Replacing the duplication was slightly painful.

### CODE REVIEW

**Comment by sentriz** [comment]
deadly, thanks brando

File: steely/plugins/gex.py [hunk 1]
```diff
@@ -304,11 +305,11 @@ def _gex_flex(bot, args, author_id, thread_id, thread_type):
     cards = gex_flex(user_id)
     total = sum(c[1] for c in cards)
     name = _user_id_to_name(bot, user_id)
-    output = '*{}*\nID: _{}_\n'.format(name, user_id)
+    output = '{}\nID: _{}_\n'.format(bold(name), user_id)
     output += 'Total cards: _{}_ (_{}_ unique)\n'.format(total, len(cards))
-    output += '\n*Cards*:\n'
+    output += f'\n{bold(Cards)}:\n'
```
Comment (iandioch) [inline steely/plugins/gex.py]: scandal

**Comment by CianLR** [comment]
Doing `from abc import *` is generally avoided 'cause it obfuscates where the objects are commitment from.
Maybe a `import formatting as fmt` would work nicer.

**Comment by CianLR** [comment]
Oops, just noticed it's merged

**Comment by iandioch** [comment]
Agree regardless, funcs and vars you can't find the definition of are the worst (especially in python)

**Comment by byxor** [comment]
Here is my preference for possible importing methods (from highest to lowest):

1. `from formatting import *` or `import formatting`
2. `import formatting as fmt`

**Pull request merged**